packages/vscode-messenger-common/src/messages.ts

```diff
@@ -4,6 +4,8 @@
  * terms of the MIT License, which is available in the project root.
  ******************************************************************************/
 
+import { CancellationToken } from './cancellation';
```

Comment:
Now there's a cyclic import between the two files. To ensure that they are harmless, at least one of the imports should be purely `import type`.

Reply:
Moved some types back